CHANGELOG.md

```diff
@@ -1,10 +1,28 @@
 # Alfred API - Changelog
 
+## 2.6.0 (2020-08-26)
+
+### Added
+* [ALFREDAPI-298](https://xenitsupport.jira.com/browse/ALFREDAPI-298): Add CRUD api for comments
+* [ALFREDAPI-403](https://xenitsupport.jira.com/browse/ALFREDAPI-403): Added support for special search terms 'isunset', 'isnull', 'isnotnull' and 'exists'
+
+### Changed
+* [ALFREDAPI-395](https://xenitsupport.jira/com/browse/ALFREDAPI-395): Change loglevels of PermissionSerivce#hasPermission
+
+### Fixed
+* [ALFREDAPI-428](https://xenitsupport.jira.com/browse/ALFREDAPI-428): Replace PersonService#getPerson(String userName) with getPersonOrNull(String userName) to avoid creation of new users when getPerson is called with a nonexistent userName.
+
+
 ## 2.5.2 (2020-08-12)
 
 ### Fixed
 * [ALFREDAPI-338](https://xenitsupport.jira.com/browse/ALFREDAPI-338): Fixed issue when the provided name would not be set while copying a node
 
+## 2.5.1 (2020-08-05)
```

Comment:
@kpiot123 This entry doesn't seem right... needs to be removed since it is part of release v2.5.2